### PR TITLE
completed prep-html-skeleton exercise

### PR DESCRIPTION
**Pull request by mahdiehmaleki**
prep-html-skeleton

### CODE REVIEW

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```
Comment (shawnkost) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md]: Don't forget about the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks good Mahdieh, just need to answer the quiz questions!

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md (hunk 1 above)
Comment (mahdiehmaleki96) [reply prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:14]: Here are the answers:
1- What tags are necessary for a complete HTML Skeleton?
<!DOCTYPE html>
<html>
<head> <title> </title> </head>
<body></body>
</html>
2- What type of content belongs within the '<head>' of an HTML document?
It contains information about the document which includes metadata (character set, title, styles (CSS), scripts).
3- What type of content belongs within the `<body>` of an HTML document?
It contains all the contents of an HTML document which is Structured content(headings, paragraphs, lists), Embedded content(images, videos, audio, iframes), and forms to collect data from users.
4- Where must the `DOCTYPE` declaration appear in a valid HTML document?
It must be the very first thing in the HTML document, before the <html> tag.

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# prep-html-skeleton-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
```
Comment (mahdiehmaleki) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:7]: <!DOCTYPE html>
<html>
<head> <title> </title> </head>
<body></body>
</html>

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- What type of content belongs within the `<head>` of an HTML document?
```
Comment (mahdiehmaleki) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:13]: It contains information about the document which includes metadata (character set, title, styles (CSS), scripts).

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+- What type of content belongs within the `<body>` of an HTML document?
```
Comment (mahdiehmaleki) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:15]: It contains all the contents of an HTML document which is Structured content(headings, paragraphs, lists), Embedded content(images, videos, audio, iframes), and forms to collect data from users.

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
```
Comment (mahdiehmaleki) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md:17]: It must be the very first thing in the HTML document, before the <html> tag.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**